src/main/java/at/ac/tuwien/kr/alpha/common/atoms/Literal.java

```diff
@@ -38,35 +38,52 @@
 import java.util.Set;
 
 /**
+ * Contains a potentially negated {@link Atom}
+ * 
  * Copyright (c) 2017-2018, the Alpha Team.
  */
-public interface Literal {
+public class Literal {
+	protected final Atom atom;
+	protected final boolean negated;
+	
+	public Literal(Atom atom, boolean negated) {
+		this.atom = atom;
+		this.negated = negated;
+	}
 
-	Atom getAtom();
+	public Atom getAtom() {
+		return atom;
+	}
 
-	boolean isNegated();
+	public boolean isNegated() {
+		return negated;
+	}
 
 	/**
 	 * Returns a new copy of this literal whose {@link Literal#isNegated()} status is inverted
 	 * (if the atom cannot be negated, the literal may be returned without creating a copy)
```

Comment:
Is the comment in parentheses still correct? It sounds a bit off, since I would rather expect an exception if a literal cannot be negated.

Reply:
No, not correct. Thanks! Fixed in 68e8f531eb2f4a74bbdab8a632b55912aa3a8f6c